src/components/booklist.js

```diff
@@ -57,7 +58,9 @@ export default function BookList({books, onHandleBooks}) {
 								</p>
 							</div>
 
-							<Button onClick={() => submit(book._id)}>Delete</Button>
+							<Button onClick={() => submit(book._id)}>
```

Comment:
Submit is not the best naming for a function ;) I see it is the modal for the delete function - maybe call it that way ;)